ssl/ssl_test.cc

```diff
@@ -11810,5 +11816,46 @@ TEST_P(PerformHybridHandshakeTest, PerformHybridHandshake) {
   }
 }
 
+// This test opens and writes a temporary pem file to test certificate file
+// loading. It is disabled by default since we may not always have permissions
+// to create files on some platforms or the generated files may encounter race
+// conditions when running multiple ssl_tests on different threads. This is
+// reenabled via all_tests.go where suitable.
```

Comment:
we have [other tests using `tmpfile()`](https://github.com/aws/aws-lc/commit/7285f2ec75bd8d8426474dccbc1674b921fe576a) that write files and haven't (to my knowledge) encountered permissions issues or race conditions. randomly generated file names will solve your concurrency issues; if `tmpfile()` doesn't resolve your permissions issues, you might be able to `mmap` your way around the issue.

Reply:
Ah thanks, I had no idea this existed! The Android issue that was mentioned in the test was the permissions issue I was running into, so this should help get past that.